3485/0/triangle.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# Triangle Project Code.
+
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+def triangle(a_side, b_side, c_side)
+  a_side, b_side, c_side = [a_side, b_side, c_side].sort
```

Comment:
Good solution. But I would say it's not the best idea to reassign method arguments.

Reply:
fixed